httpserver/http_connection.cpp

```diff
@@ -589,48 +624,49 @@ void connection_t::poll_db(std::string pk, std::string last_hash) {
         response_.set(http::field::content_type, "text/plain");
         BOOST_LOG_TRIVIAL(error)
             << "Internal Server Error. Could not retrieve messages for "
-            << pk.substr(0, 2) << "..."
-            << pk.substr(pk.length() - 3, pk.length() - 1);
+            << obfuscate_pubkey(pk);
         return;
     }
 
     const bool lp_requested =
         request_.find("X-Loki-Long-Poll") != request_.end();
 
-    if (items.empty() && lp_requested && long_polling_counter < DB_POLL_LIMIT) {
+    if (!items.empty()) {
+        BOOST_LOG_TRIVIAL(trace) << obfuscate_pubkey(pk);
```

Comment:
Is this meant to be started with `"Successfully retrieved messages for "`?

Reply:
yep, will edit